articles/integrations/aws.md

```diff
@@ -271,7 +271,7 @@ The result of calling the delegation endpoint will contain the AWS token in the
 }
 ```
 
-**NOTE:** The Auth0 client libraries simplify calling these endpoints. See our [GitHub repo](https://github.com/auth0/) for the latest SDKs. See an example for client-side JavaScript at: [Delegation Token Request](https://github.com/auth0/auth0.js#delegation-token-request).
+**NOTE:** The [Auth0 client libraries](/libraries) simplify the process of calling these endpoints. See an example for client-side JavaScript at: [Delegation Token Request](https://github.com/auth0/auth0.js#delegation-token-request).
```

Comment:
In auth0.js v8 the getDelegationToken method no longer exists. There is a `delegation` method, but it's deprecated and so wasn't to be documented.

https://auth0.com/docs/libraries/auth0js/v7#delegation-token-request describes what you want, but if you're going to link to that, you should make some kind of note about how that's auth0.js v7, and how delegation is a deprecated concept in v8, I guess?

I think if it were me I'd just remove that entire block of text, for now, and the sample below it exhibiting it, maybe.